apps/innovations/_services/innovation-supports.service.ts

```diff
@@ -952,6 +954,11 @@ export class InnovationSupportsService extends BaseService {
       return { id: savedSupport.id, newAssignedAccessors: new Set(newAssignedAccessors), threadId: thread.thread.id };
     });
 
+    // Create satisfaction survey if the support was closed
+    if (data.status === InnovationSupportStatusEnum.CLOSED) {
```

Comment:
This should probably be inside the transaction

Reply:
Moved it.